man/faq/developer/reference-compatibility.Rmd

```diff
@@ -0,0 +1,42 @@
+
+vctrs provides a framework for working with vector classes in a generic way. However, it implements several compatibility fallbacks to base R methods. In this reference you will find how vctrs tries to be compatible with your vector class, and what base methods you need to implement for compatibility.
```

Comment:
(If you're starting from scratch, we think you'll find it easier to start using `new_vctr()`; this guide is aimed developers with existing vector classes)